packages/scripts/turbo/generators/config.ts

```diff
@@ -71,7 +70,7 @@ export default function generator(plop: PlopTypes.NodePlopAPI): void {
 				path: `${plop.getDestBasePath()}/../../.github/labeler.yml`,
 				transform(content, answers) {
 					const labelerYAML = parseYAML(content) as Record<string, string[]>;
-					labelerYAML[`packages:${answers.name}`] = [`packages/${answers.name}/*`, `packages/${answers.name}/**/*`];
+					labelerYAML[`package:${answers.name}`] = [`package/${answers.name}/*`, `package/${answers.name}/**/*`];
```

Comment:
```suggestion
					labelerYAML[`package:${answers.name}`] = [`packages/${answers.name}/*`, `packages/${answers.name}/**/*`];
```

These paths stay the same??

Reply:
Actually, both are wrong due to #10272. I'll commit this suggestion, but fixing this would ideally be done in another pull request. It's pretty low priority.

Reply:
ah, right

Reply:
#10332